buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -113,7 +113,8 @@ public void apply(Project project) {
         for (ElasticsearchDistribution distribution : testDistributions) {
             String taskname = destructiveDistroTestTaskName(distribution);
             TaskProvider<?> depsTask = project.getTasks().register(taskname + "#deps");
-            depsTask.configure(t -> t.dependsOn(distribution, examplePlugin, quotaAwareFsPlugin));
+            // explicitly depend on the archive not on the implicit extracted distribution
+            depsTask.configure(t -> t.dependsOn(distribution.getArchiveBuildDependencies(), examplePlugin, quotaAwareFsPlugin));
```

Comment:
Looking at this more, now that we make the default behavior to return the extracted archive, perhaps instead of exposing a `getExtracted()` method on `ElasticsearchDistribution` we should replace that with `getArchive()`?

Reply:
sounds like a good idea